examples/plot_map_in_3d.py

```diff
@@ -41,4 +41,4 @@
                 frame='heliocentricinertial')
```

Comment:
I can't add a comment to the png above, but should that be here?

Reply:
Ah, sorry, that too.